auto_py_to_exe/web/css/general.css

```diff
@@ -1,6 +1,6 @@
 @font-face {
-    font-family: 'Nunito';
-    src: url('../Nunito-Light.ttf');
```

Comment:
If we are not using this .ttf file anymore, we might as well remove the file.

Reply:
Sure, I just kept it in case we wanted to bring it back.